pagespeed/envoy/exception.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I don't think there's any code in mod_pagespeed that throws, do we need this?

Reply:
remove this file
code pushed in https://github.com/apache/incubator-pagespeed-mod/pull/1969/commits/4ab87afdc150b403bcef176c8f4934e8f4ba18d1